vector/src/main/java/im/vector/app/features/MainActivity.kt

```diff
@@ -241,7 +241,7 @@ class MainActivity : VectorBaseActivity<ActivityMainBinding>(), UnlockedActivity
                 // We have a session.
                 // Check it can be opened
                 if (sessionHolder.getActiveSession().isOpenable) {
-                    HomeActivity.newIntent(this)
+                    HomeActivity.newIntent(this, existingSession = true)
```

Comment:
Here we want to invoke threads migration (clear cache) only if there is no initial sync already applied. For example when a user login for the first time existing session will be false, so no migration will happen